docs/v3/source/includes/concepts/_lifecycles.md.erb

```diff
@@ -62,3 +62,28 @@ Name | Type | Description
 **type** | _string_ | `docker`
 **data** | _object_ | Data is not used by the Docker lifecycle; valid value is `{}`
 
+### Kpack lifecycle
+
+```
+Example Kpack Lifecycle
+```
+
+```json
+{
+  "type": "kpack",
+  "data": {
+    "buildpacks": ["paketo-buildpacks/java"]
+  }
+}
+```
+
+This is the default lifecycle for Cloud Foundry for Kubernetes. When staging an app with this lifecycle, the app source code will be
+compiled using a [Cloud Native Buildpack](https://buildpacks.io/) by [kpack](https://github.com/pivotal/kpack), resulting in an OCI image.  When running an app with this lifecycle,
+a container is created and the OCI image is executed inside of it.
```

Comment:
might worth mentioning that the build creates a `docker` lifecycle droplet

Reply:
I didn't even realize droplets had a reference to a `lifecycle`. That's a bit surprising to me that they're `docker` so this sounds like a good thing to document.

For the curious, this is an example droplet that resulted from `kpack` staging:
```json
{
   "guid": "7f019d25-684b-4fdb-a380-390d70a87701",
   "created_at": "2020-09-02T19:14:40Z",
   "updated_at": "2020-09-02T19:15:05Z",
   "state": "STAGED",
   "error": null,
   "lifecycle": {
      "type": "docker",
      "data": {}
   },
   "checksum": null,
   "buildpacks": null,
   "stack": null,
   "image": "gcr.io/cf-capi-arya/cf-workloads/397fe53b-9eb6-49c5-a323-9fe675fe2ede@sha256:5964557c69b246acf98903954d82e8716910579758905971e4b9da0219b72f5f",
   "execution_metadata": null,
   "process_types": {
      "web": "bundle exec rackup config.ru -p $PORT",
      "worker": "bundle exec rackup config.ru"
   },
   "relationships": {
      "app": {
         "data": {
            "guid": "397fe53b-9eb6-49c5-a323-9fe675fe2ede"
         }
      }
   },
   "metadata": {
      "labels": {},
      "annotations": {}
   },
   "links": {
      "self": {
         "href": "https://api.tim.k8s.capi.land/v3/droplets/7f019d25-684b-4fdb-a380-390d70a87701"
      },
      "app": {
         "href": "https://api.tim.k8s.capi.land/v3/apps/397fe53b-9eb6-49c5-a323-9fe675fe2ede"
      },
      "assign_current_droplet": {
         "href": "https://api.tim.k8s.capi.land/v3/apps/397fe53b-9eb6-49c5-a323-9fe675fe2ede/relationships/current_droplet",
         "method": "PATCH"
      },
      "package": {
         "href": "https://api.tim.k8s.capi.land/v3/packages/c413a333-ac4d-4c9b-8402-e74469f8c137"
      }
   }
}
```

Reply:
I pushed some changes! 🙏